packages/color-mode/src/color-mode-script.tsx

```diff
@@ -1,50 +1,17 @@
 import * as React from "react"
-import { ColorMode } from "./color-mode-provider"
 
-type Mode = ColorMode | "system" | undefined
-
-function setColorModeVar(initialValue: Mode) {
-  const mql = window.matchMedia("(prefers-color-scheme: dark)")
-  const systemPreference = mql.matches ? "dark" : "light"
-
-  let persistedPreference: Mode
-
-  try {
-    persistedPreference = localStorage.getItem("chakra-ui-color-mode") as Mode
-  } catch (error) {
-    console.log(
-      "Chakra UI: localStorage is not available. Color mode persistence might not work as expected",
-    )
-  }
-
-  const isInStorage = typeof persistedPreference === "string"
-
-  let colorMode: Mode
-
-  if (isInStorage) {
-    colorMode = persistedPreference
-  } else {
-    colorMode = initialValue === "system" ? systemPreference : initialValue
-  }
-
-  if (colorMode) {
-    const root = document.documentElement
-    root.style.setProperty("--chakra-ui-color-mode", colorMode)
-  }
-}
-
-interface ColorModeScriptProps {
-  defaultColorMode?: ColorMode
-}
+const noFlash = `(function() { try {
```

Comment:
do we still need this? In the old commit from `rc.2` I found it used in the gatsby plugin - but since we never check for any `className` and only sync ourselves, I wonder what its purpose was?